01-big-shiny.Rmd

```diff
@@ -8,7 +8,6 @@
 
 If you are reading this page, chances are you already know what a `{shiny}` application (sometimes shortened as "`{shiny}` app") is—**a web application that communicates with R, built in R, and working with R**.
 The beauty of `{shiny}` [@R-shiny] is that it makes it easy for someone already familiar with R to create a small app in a matter of hours.
-With small and minimal `{shiny}` apps, no knowledge of HTML (HyperText Markup Language), CSS(Cascading Style Sheets) or JavaScript is required, and you do not have to think about technical elements that usually come with web applications—for example, you do not have to think about the port the application is served on: `{shiny}` picks one for you.[^big-shiny-1]
```

Comment:
Hey, 

Why is this line deleted ?

Reply:
Sorry, I did not mean to delete it! It was a mistake

Reply:
No problem :) 

can you re-push the old line please? Thanks :)